atomicfu/src/commonMain/kotlin/kotlinx/atomicfu/AtomicFU.common.kt

```diff
@@ -23,6 +23,9 @@ import kotlin.reflect.KProperty0
  */
 public expect fun <T> atomic(initial: T, trace: TraceBase = None): AtomicRef<T>
 
+// Binary compatibility
+public expect fun <T> atomic(initial: T): AtomicRef<T>
```

Comment:
Now, these functions have missing javadoc, should be fixed

Reply:
Not what I've meant :)

This comment is required _for us_ in order to maintain the library. It should be one-liner or `/* */` block in the start of the file.

Javadoc is for library users. It probably should say `Creates atomic reference with a given [initial] value.`.
And Javadoc for `atomic(initial: T, trace: TraceBase = None)` should also explain what trace parameter is